### PR TITLE
Ember A11y 2016-06-01

### PR DESCRIPTION
**Pull request by nathanhammond**
Meeting notes! Respond with reactions for approval, or corrections as comments.

/cc @ember-a11y/core

### CODE REVIEW

File: ember-a11y/2016-06/june-01.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  - Lots of followup questions as we hadn't made things clear.
+  - Maintainers took time immediately, but that's a lot to ask for.
+- Project night at Ember London.
+  - Tentative about making the entire night about a11y.
+  - One of the best-attended project nights they've had.
+  - People seemed to be thrilled about taking on a11y projects.
+  - Most people were very excited to get started.
+  - George and Blaze were in attendance and brought a JAWS machine.
+    - [Tracked down and submitted an IE fix to Ember Observer.](https://github.com/emberobserver/client/pull/37)
+    - [Fixed an aria-hidden bug in ember-tooltips.](https://github.com/emberobserver/client/pull/37)
```
Comment (sarenji) [inline ember-a11y/2016-06/june-01.md]: wrong link, would love to see the PR!

Comment (jgwhite) [inline ember-a11y/2016-06/june-01.md]: https://github.com/sir-dunxalot/ember-tooltips/pull/64

File: ember-a11y/2016-06/june-01.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+- Outreach
+  - Very positive response from addon community and maintainers.
+  - Lots of followup questions as we hadn't made things clear.
+  - Maintainers took time immediately, but that's a lot to ask for.
+- Project night at Ember London.
+  - Tentative about making the entire night about a11y.
+  - One of the best-attended project nights they've had.
+  - People seemed to be thrilled about taking on a11y projects.
+  - Most people were very excited to get started.
+  - George and Blaze were in attendance and brought a JAWS machine.
```
Comment (jgwhite) [inline ember-a11y/2016-06/june-01.md]:
``` diff
- George and Blaze
+ Ben and George from Blazie (accessibility consultants)
```

Comment (nathanhammond) [inline ember-a11y/2016-06/june-01.md]: Whoa, clever use of markdown. (Takes notes.)

Comment (jgwhite) [inline ember-a11y/2016-06/june-01.md]: Haha, why thank you! 😊

**Pull request merged**